Makefile

```diff
@@ -118,12 +117,15 @@ else
 endif
 
 test-backend-unit:
-	cd lumigator/python/mzai/backend/backend/tests; \
+	cd lumigator/python/mzai/backend/; \
+	rm local.db; \
```

Comment:
will this break folks' current setup? we might want to add some warning docs here

Reply:
I'll add a warning, but basically one shouldn't reuse the local db unless one knows what they're doing. It actually causes issues because people tend to forget removing this before running tests, right now.